app/assets/stylesheets/custom.css.scss

```diff
@@ -44,6 +44,45 @@ body {
   }
 }
 
+.smfont {
+  font-size: 0.9em;
+}
+.warning {
+  color: red;
+  padding-left: 30px;
+}
+
+.card {
+  background-color: #FFF;
+  border: 1px solid transparent;
+  border-color: #BCE8F1;
+  border-radius: 4px;
+  box-shadow: 0px 1px 1px rgba(0, 0, 0, 0.05);
+}
+
+.card__heading {
+  padding: 10px 15px;
+  border-bottom: 1px solid transparent;
+  border-top-right-radius: 3px;
+  border-top-left-radius: 3px;
+}
+.card__heading_new {
+  color: #fff;
+  background-color: #5bc0de;
+  border-color: #BCE8F1;

```

Comment:
Color literals like `#BCE8F1` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#BCE8F1` should be written as `#bce8f1`